Common/GPU/Vulkan/thin3d_vulkan.cpp

```diff
@@ -825,6 +825,14 @@ VKContext::VKContext(VulkanContext *vulkan, bool splitSubmit)
 		bugs_.Infest(Bugs::EQUAL_WZ_CORRUPTS_DEPTH);
 		// At least one driver at the upper end of the range is known to be likely to suffer from the bug causing issue #13833 (Midnight Club map broken).
 		bugs_.Infest(Bugs::MALI_STENCIL_DISCARD_BUG);
+
+		// This started in driver 31 or 32.
+		if (VK_API_VERSION_MAJOR(deviceProps.driverVersion) >= 32) {
+			NOTICE_LOG(G3D, "Driver version %08x infested: major=%d", deviceProps.driverVersion, VK_API_VERSION_MAJOR(deviceProps.driverVersion));
+			bugs_.Infest(Bugs::MALI_CONSTANT_LOAD_BUG);  // See issue #15661
+		} else {
+			NOTICE_LOG(G3D, "Driver version %08x not infested: major=%d", deviceProps.driverVersion, VK_API_VERSION_MAJOR(deviceProps.driverVersion));
```

Comment:
nit: Seems slightly noisy, but probably fine.

-[Unknown]

Reply:
Oops, meant to delete these.